pkg/models/tag.go

```diff
@@ -0,0 +1,12 @@
+package models
+
+import (
+	"github.com/google/uuid"
+)
+
+// Tag represents a reference to another data structure in the database
+type Tag struct {
+	BaseStructUser
```

Comment:
I don't think this needs to implement `BaseStructUser` necessarily, every field in it right now is empty